prometheus-exporters/vrops-exporter/alerts/vccluster.alerts

```diff
@@ -74,3 +74,17 @@ groups:
     annotations:
       description: "VC {{ $labels.vcenter }} {{ $labels.vccluster }} has NO failover host amount configured, this should be 1"
       summary: "VC {{ $labels.vcenter }} {{ $labels.vccluster }} has NO failover host amount configured, this should be 1"
+
+  - alert: ClusterDRSNotFullyAutomated
+      expr: vrops_cluster_configuration_drsconfig_defaultvmbehavior{state != "fullyAutomated"} == 0
```

Comment:
prom rule listing fails. Maybe remove the spaces. 

`79 prometheus-exporters/vrops-exporter/alerts/vccluster.alerts: yaml: line 79: mapping values are not allowed in this context`

Reply:
Don't know how that slipped through